UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/MatrixSystemManager.cs

```diff
@@ -15,12 +16,10 @@ public class MatrixSystemManager : MonoBehaviour
 
 	public void SelfInitialize()
 	{
-		StartCoroutine(Initialize());
+		_ = Initialize();
 	}
 
-
-
-	public IEnumerator Initialize()
+	public async UniTaskVoid Initialize()
```

Comment:
This needs rigourous testing, have you tested with Build and headless? and copy and paste

Reply:
something so major I think needs an headless test to

Reply:
tested on headless, works just fine.